campaign-launcher/interfaceV2/src/components/Account/index.tsx

```diff
@@ -88,23 +115,22 @@ const Account: FC = () => {
           },
         }}
       >
-        <Button
-          onClick={handleLogout}
-          sx={{
-            color: 'primary.light',
-            p: 1,
-            width: '100%',
-            fontWeight: 600,
-            justifyContent: 'flex-start',
-            gap: 1,
-            '&:hover': {
-              backgroundColor: 'unset',
-            },
-          }}
-        >
-          <PowerIcon />
-          Disconnect wallet
-        </Button>
+        <List sx={{ p: 0 }}>
```

Comment:
<img width="474" height="350" alt="Screenshot 2025-07-31 at 14 28 52" src="https://github.com/user-attachments/assets/16c75e0b-cf3b-4bae-ab3a-30c8fc410135" />
Maybe let's try to have these items aligned?